CatAnalyzer/plugins/ttbbLepJetsAnalyzer.cc

```diff
@@ -251,63 +319,114 @@ ttbbLepJetsAnalyzer::ttbbLepJetsAnalyzer(const edm::ParameterSet& iConfig):
   tree->Branch("run",        &b_Run,         "Run/I");
   tree->Branch("luminumber", &b_Lumi_Number, "Lumi_Number/I");
   tree->Branch("genweight",  &b_GenWeight,   "GenWeight/F");
-
-  tree->Branch("PUWeight", "std::vector<float>", &b_PUWeight);
-  tree->Branch("GoodPV",   &b_nGoodPV,  "nGoodPV/I");
+  tree->Branch("PUWeight",   "std::vector<float>", &b_PUWeight);
+  tree->Branch("GoodPV",     &b_nGoodPV,  "nGoodPV/I");
 
   tree->Branch("channel",  &b_Channel,  "Channel/I");
 
   tree->Branch("MET",     &b_MET,     "MET/F");
   tree->Branch("MET_phi", &b_MET_phi, "MET_phi/F");
 
-  tree->Branch("lepton_px", &b_Lepton_px, "lepton_px/F");
-  tree->Branch("lepton_py", &b_Lepton_py, "lepton_py/F");
-  tree->Branch("lepton_pz", &b_Lepton_pz, "lepton_pz/F");
-  tree->Branch("lepton_E" , &b_Lepton_E,  "lepton_E/F" );
-  tree->Branch("lepton_pT", &b_Lepton_pT, "lepton_pT/F" );
-
+  tree->Branch("lepton_px",  &b_Lepton_px, "lepton_px/F");
+  tree->Branch("lepton_py",  &b_Lepton_py, "lepton_py/F");
+  tree->Branch("lepton_pz",  &b_Lepton_pz, "lepton_pz/F");
+  tree->Branch("lepton_E" ,  &b_Lepton_E,  "lepton_E/F" );
+  tree->Branch("lepton_pT",  &b_Lepton_pT, "lepton_pT/F" );
   tree->Branch("lepton_SF",  "std::vector<float>", &b_Lepton_SF );
   tree->Branch("lepton_LES", &b_Lepton_LES, "lepton_LES/F" );
 
-  tree->Branch("jet_px", "std::vector<float>", &b_Jet_px);
-  tree->Branch("jet_py", "std::vector<float>", &b_Jet_py);
-  tree->Branch("jet_pz", "std::vector<float>", &b_Jet_pz);
-  tree->Branch("jet_E" , "std::vector<float>", &b_Jet_E );
-  tree->Branch("jet_pT", "std::vector<float>", &b_Jet_pT );
-
+  tree->Branch("jet_px",     "std::vector<float>", &b_Jet_px);
+  tree->Branch("jet_py",     "std::vector<float>", &b_Jet_py);
+  tree->Branch("jet_pz",     "std::vector<float>", &b_Jet_pz);
+  tree->Branch("jet_E" ,     "std::vector<float>", &b_Jet_E );
```

Comment:
Wouldn't it be more useful to have pt, eta, phi information instead of px,py,pz?